source/develop/release-management/features/storage/cinderlib-integration.html.md

```diff
@@ -267,6 +267,32 @@ Cinderlib will be packaged in RPM.
 The Cinder drivers and Cinderlib needs to be installed in the Engine.
 OS-Brick needs to installed on the hosts
 
+#### Manual Install
+
+Since that some of the required libraries are currently not part of oVirt installation, the following steps are required:
+
+On Engine:
+
+```
+$ yum install -y centos-release-openstack-pike
+$ yum-config-manager --enable openstack-pike
+$ yum install -y openstack-cinder python-pip
+$ pip install cinderlib
+```
+
+On all the hosts: (Vdsm needs to be restarted after installing the packages)
+
+```
+$ yum install -y centos-release-openstack-pike
+$ yum install -y python2-os-brick
```

Comment:
Don't we need to enable the repo like we do on engine?

If we have packages in yum, why we don't require them in vdsm?

Reply:
The package is available in yum only with the openstack repo installed.
We will need to have them available via oVirt channel.